diff --git a/src/pages/Dashboard.jsx b/src/pages/Dashboard.jsx
index 91c7dc5..54e199e 100644
--- a/src/pages/Dashboard.jsx
+++ b/src/pages/Dashboard.jsx
@@ -41,7 +41,7 @@ const Dashboard = () => {
       try {
         setLoadingPlants(true)
         const res = await api.get('/plants/user-plants/me')
-        const mapped = res.data.map(p => ({
+        const mapped = res.data.data.map(p => ({
           id:       p.id,
           emoji:    p.emoji  || '🪴',
           name:     p.name,
diff --git a/src/pages/Login.jsx b/src/pages/Login.jsx
index 3ff97fe..9d46df1 100644
--- a/src/pages/Login.jsx
+++ b/src/pages/Login.jsx
@@ -53,10 +53,10 @@ const Login = () => {
       })
 
       const userData = {
-        id:    res.data.id,
-        name:  res.data.name,
-        phone: res.data.phone,
-        token: res.data.token,
+        id:    res.data.data.id,
+        name:  res.data.data.name,
+        phone: res.data.data.phone,
+        token: res.data.data.token,
       }
 
       login(userData)
diff --git a/src/pages/Register.jsx b/src/pages/Register.jsx
index 50e498f..5065642 100644
--- a/src/pages/Register.jsx
+++ b/src/pages/Register.jsx
@@ -54,10 +54,10 @@ const Register = () => {
       })
 
       const userData = {
-        id:    res.data.id,
-        name:  res.data.name,
-        phone: res.data.phone,
-        token: res.data.token,
+        id:    res.data.data.id,
+        name:  res.data.data.name,
+        phone: res.data.data.phone,
+        token: res.data.data.token,
       }
 
       login(userData)
